src/list_structure.js

```diff
@@ -49,3 +49,17 @@ function addTodo(todo) {
     }
 }
 
+function updateLS() {
+    todosEl = document.querySelectorAll('li')
+
+    const todos = []
+
+    todosEl.forEach(todoEl => {
+        todos.push({
+            text: todoEl.innerText,
+            completed: todoEl.classList.contains('completed')
+        })
+    })
+
+    localStorage.setItem('todos', JSON.stringify(todos))
```

Comment:
- [ ] The add and remove functionalities are not part of this project, You will implement that in the upcoming project. Please remove this functionality and create some hard-coded simple to-do tasks and display them on the screen.